pkg/consts/qos.go

```diff
@@ -37,6 +37,8 @@ const (
 
 	PodAnnotationMemoryEnhancementNumaExclusive       = "numa_exclusive"
 	PodAnnotationMemoryEnhancementNumaExclusiveEnable = "true"
+
+	PodAnnotationMemoryEnhancementOOMPriority = "oom_priority"
```

Comment:
please add comments describing this enhancement and the valid values 


ps: other collaborators will also supplement comments for other enhancements

Reply:
done

Reply:
done